Completed Readme second draft (#23)
UBC-MDS/game_sales: README.md

> # game_sales Dashboard

Reviewer: ```suggestion
# Game Sales Dashboard
```

> game_sales is an interactive dashboard that allows users to easily visualize and filter high level sales information from the video games industry.

Reviewer: ```suggestion
Game Sales is an interactive dashboard that allows users to easily visualize and filter high level sales information from the video games industry.
```

- Multiple selection box that allows the user to change the number of publishers considered in the ranking of the bar plot.

Users will be able to observe the evolution of sales of the video game industry and get insights into its trends by genre, publisher and region.

# Usage

## Online

> Using our app online is as easy as clicking the Heroku [link](https://game-sales-dash-app.herokuapp.com/)!

Reviewer: ```suggestion
Using our app online is as easy as clicking the Heroku [link](https://dsci-532-2022-gamesales.herokuapp.com/)!
```

## Local

In order to run our app locally please follow these steps:
1. Clone the [repository](https://github.com/UBC-MDS/game_sales.git) to your local machine.
2. Navigate to the cloned repository within your console.

> 3. Run the command `python app.py`, this should launch the app in a local web browser.

Reviewer: ```suggestion
3. Run the command `python src/app.py`, this should launch the app in a local web browser.
```
